.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+## Description
+*Please describe what changes you made, and why you feel they are necessary. Make sure to include
+code examples, where applicable.*
+
+Corresponding issue:
+
+## Testing
+*Please explain how you tested this change manually, and, if applicable, what new tests you added. If
+you're making a change to just the website, you can omit this section.*
+
+## Checklist
+
+* [ ] I have read the contribution guidelines
+* [ ] I have targeted this PR against the correct branch (`master` for website changes, `dev` for
+  source changes)
+* [ ] This is either a bugfix, a documentation update, or a new feature that has been explicitly
```

Comment:
When do I have to select this? Linking an issue should be enough, shouldn't it?

Reply:
We want people to know what kind of PRs we're looking for, which is what's described here.